hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestRegionServerOnlineConfigChange.java

```diff
@@ -259,4 +263,12 @@ public void removeClosedRegionFromConfigurationManager() throws Exception {
       });
     }
   }
+
+  @Test
+  public void testStoreConfigurationOnlineChange() {
+    rs1.getConfigurationManager().notifyAllObservers(conf);
+    long actualMaxFileSize =
+        r1.getStore(COLUMN_FAMILY1).conf.getLong(TableDescriptorBuilder.MAX_FILESIZE, -1);
```

Comment:
Strange that we do not provide getConfiguration method for Region and Store interface...

Let's open another issue to address it.